src/app/home/upcoming-events/upcoming-events.component.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Component,
+  OnInit,
+  Input,
+  CUSTOM_ELEMENTS_SCHEMA,
+} from '@angular/core';
+import { register } from 'swiper/element/bundle';
+import { IonicSlides } from '@ionic/angular';
+import { CommonModule } from '@angular/common';
+register();
+
+@Component({
+  selector: 'app-upcoming-events',
+  templateUrl: './upcoming-events.component.html',
+  styleUrls: ['./upcoming-events.component.scss'],
+  standalone: true,
```

Comment:
Why using standalone components? We agreed not to use them, so we must be consistent

Reply:
Yes, my falt. I was looking one demo with standalone component